Detectors/TPC/calibration/src/TPCFastSpaceChargeCorrectionHelper.cxx

```diff
@@ -356,11 +374,13 @@ void TPCFastSpaceChargeCorrectionHelper::testGeometry(const TPCFastTransformGeo&
   }
 }
 
-std::unique_ptr<o2::gpu::TPCFastSpaceChargeCorrection> TPCFastSpaceChargeCorrectionHelper::createSpaceChargeCorrection(
+std::unique_ptr<o2::gpu::TPCFastSpaceChargeCorrection> TPCFastSpaceChargeCorrectionHelper::createFromTrackResiduals(
   const o2::tpc::TrackResiduals& trackResiduals, TTree* voxResTree)
```

Comment:
I propose to implement options to switch between smoothed and non-smoothed and also to be able to invert signs:

```suggestion
std::unique_ptr<o2::gpu::TPCFastSpaceChargeCorrection> TPCFastSpaceChargeCorrectionHelper::createFromTrackResiduals(
  const o2::tpc::TrackResiduals& trackResiduals, TTree* voxResTree, bool useSmoothed, bool invertSigns)
```

Reply:
Hi @wiechula,  
I have put your changes in another PR: https://github.com/AliceO2Group/AliceO2/pull/11731
since this one is already merged